desktop/core/src/desktop/lib/rest/raz_http_client.py

```diff
@@ -45,7 +45,9 @@ def execute(self, http_method, path, params=None, data=None, headers=None, allow
 
     response = raz_client.get_url(action=http_method, path=url, headers=headers)
 
-    signed_url = url + ('?' if '?' not in url else '&') + response['token']
+    signed_url = url
+    if response.get('token') is not None:
```

Comment:
nit: logically could even be just 

`if response.get('token')`

Reply:
I usually follow some of them like https://www.python.org/dev/peps/pep-0008/#programming-recommendations.
2nd point ;)  to not have out the box crashes haha!

Reply:
What the point of potentially appending ''?